frontend/src/assets/css/styles.scss

```diff
@@ -68,3 +68,26 @@ code {
     transform: translateY(0px)
   }
 }
+
+// login
```

Comment:
The style for authorization should be moved to the scss module file.
Example - frontend/src/components/counter/styles.module.scss

Reply:
Decided to use BEM